core/test/qa_Block.cpp

```diff
@@ -22,6 +22,7 @@ struct BlockSignaturesNone : public gr::Block<BlockSignaturesNone<T>> {
 };
 
 ENABLE_REFLECTION_FOR_TEMPLATE(BlockSignaturesNone, in, out);
+// NOTE: If you move BlockSignaturesNone to blocklib, call GR_REGISTER_BLOCK
```

Comment:
Superfluous comment in this case, as these prototype blocks are only empty implementations needed for their `process[Bulk, One](...)` function signatures. Comments can be removed as these blocks will never make it into the blocklib.